### ISSUE TITLE
Toc in sidebar has no styling for >4 header levels

### ISSUE THREAD

**Issue by abchenson** [readthedocs/sphinx_rtd_theme#540]
This is related to

https://github.com/rtfd/sphinx_rtd_theme/issues/539

Having set ```'navigation_depth': 10``` in html_theme_options in conf.py, I can now get 4th level and above titles in the side menu table of contents.  But I notice they behave quite differently after the third layer.  Is this deliberate?
Details: At forth level I see something like this when looking at "Page A":
```
+ First
    + Second
        +Third
            +Page A
Titles On Page A
...
            +Page B
Titles On Page B
...
```
That is, the titles after level 3 are not collapsed properly (they are all open and cannot be collapsed) and are not properly indented.  If I remove a level everything looks fine, only the titles for the open page are opened and the indent is correct.

**Comment by jessetan** [comment]
I think the stylesheet is only written for 4 levels of header in the toctree.

**Comment by abchenson** [comment]
OK - We have restyled the menu in our custom theme to add a level.  Although ideally it's nicest to keep the menu fairly flat, it is occasionally useful to us to have the 5th level.

**Comment by Blendify** [comment]
@jessetan is there an easy way that we can style this to the Nth level? Some trickery with css selectors should work.

**Comment by ederag** [comment]
After the 4th level the indentation could start over from zero.
The font size being smaller, there would be no ambiguity.
No need to further reduce the font size after the 5th level.

**Comment by agjohnson** [comment]
I don't support this many navigation levels as a concept, it will be very difficult for your readers to use your navigation. I'm on the fence whether I even want to endorse this ability with this theme, where our theme focus is readability for the user.

If we addressed this at all, I agree that continuing indent is the best visual designation, though I'd say font size and color could remain the same as l4. L4 TOC is already not legible or up to a11y standards of contrast or size, so further changes will be even more against our theme goals.

**Comment by ederag** [comment]
i do agree that the font size could be the same as L4, since the indentation removes any ambiguity.
It is also true that for the main documentation more than 4 levels is usually not advisable.

Yet it would be useful for auto-generated documentation of source code:
https://geoptics.readthedocs.io/en/latest/geoptics.html#subpackages
For instance right now it is impossible to quickly switch between
`geoptics.elements.rays` and `geoptics.guis.qt.rays` modules documentation.

**Comment by agjohnson** [comment]
Ah true. It helps to know the use cases for these sorts of requests -- this seems like a valid one that we could support with some default style rules for level + N nesting. Indentation only seems like a good middle ground.

**Comment by lulubell** [comment]
In our case we have a looooot of pages with many subsections. People who use it already know the topics on a high levels, but they search for help on lower levels. So they know, what they search for, but it's easier when it's organised in bigger topics. There is no way to split it to smaller pages and keep it readable and easy to use.

I would use t3SphinxThemeRtd, as it handles this perfectly, but there are other consequences of such decision. So it would be really nice to know, if you are planning to add this to the next release.

**Comment by arnons1** [comment]
Joining here. Also interested, primarily in Level 5 ToC.
I use Level 1 as very rough categories. Sometimes it just makes a lot more sense to split documents up as sub-documents (for example, Linux and Windows instructions separately), and that causes Level 5 to show up which really doesn't work well with the sidebar :(

**Comment by louking** [comment]
I, too, just found this looking for level 5 ToC in navigation bar. I see #852 was merged recently. Do we know when this will be available on PyPI?

**Comment by Blendify** [comment]
This has been fixed with ca015a57042713ebd7f39563cdb01fab5b604051
